org.thentrythis.Samplebrain.yaml

```diff
@@ -0,0 +1,97 @@
+id: org.thentrythis.Samplebrain
+runtime: org.kde.Platform
+runtime-version: "5.15-22.08"
+sdk: org.kde.Sdk
+
+rename-desktop-file: samplebrain.desktop
+rename-icon: samplebrain
+
+command: samplebrain
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+
+cleanup:
+  - "/include"
+  - "/lib/cmake"
+  - "/lib/pkgconfig"
+  - "/share/doc"
+
+modules:
+  - name: libsndfile
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+      - "-DBUILD_SHARED_LIBS=ON"
+      - "-DENABLE_EXTERNAL_LIBS=NO"
+      - "-DENABLE_CPACK=NO"
+      - "-DBUILD_PROGRAMS=OFF"
+      - "-DBUILD_EXAMPLES=OFF"
+    sources:
+      - type: archive
+        url: https://github.com/libsndfile/libsndfile/releases/download/1.1.0/libsndfile-1.1.0.tar.xz
+        sha256: 0f98e101c0f7c850a71225fb5feaf33b106227b3d331333ddc9bacee190bcf41
+
+  - name: portaudio
+    buildsystem: autotools
+    config-opts:
+      - "--with-alsa"
+      - "--without-jack"
+      - "--without-oss"
+      - "--without-asihpi"
+    sources:
+      - type: archive
+        url: http://files.portaudio.com/archives/pa_stable_v190700_20210406.tgz
+        sha256: 47efbf42c77c19a05d22e627d42873e991ec0c1357219c0d74ce6a2948cb2def
+
+  - name: liblo
+    buildsystem: autotools
+    config-opts:
+      - "--disable-tests"
+      - "--disable-network-tests"
+      - "--disable-tools"
+      - "--disable-examples"
+    sources:
+      - type: archive
+        url: https://versaweb.dl.sourceforge.net/project/liblo/liblo/0.31/liblo-0.31.tar.gz
+        sha256: 2b4f446e1220dcd624ecd8405248b08b7601e9a0d87a0b94730c2907dbccc750
+
+  - name: samplebrain
+    buildsystem: qmake
+    builddir: true
+    build-options:
+      env:
+        PREFIX: /app
+    config-opts:
+      - "LIBS+=-L/app/lib"
+    sources:
+      - type: git
+        url: https://gitlab.com/then-try-this/samplebrain.git
+        tag: 0.18_release
+        commit: 4e36c9292334729b895d7353468b9acd9c085c1b
+      ## build: allow setting installation prefix
+      - type: patch
+        path: patches/01_74b00570ddee65d134a7d95f87c11b2458a64821.patch
+      ## linux: don't use hardcoded path for Exec and Icon
```

Comment:
you can just run `desktop-file-edit` for that. Also `rename-icon` takes care of the icon name.

Reply:
All the patches are unmodified commits from upstream (or my pending merge request). I'd rather not change them, as using them like this means I can remove them all with no other modifications to the manifest when there's a new release that includes them.

Is it okay to use patches for this?

Reply:
Yes.